deploy/README.md

```diff
@@ -0,0 +1,26 @@
+# Deployment instructions
+
+Vote is meant to be deployed on an OKD/Openshift cluster.
+This directory contains most of the files needed to do that, but doesn't include the config maps, since they contain secrets.
+You'll need to create 2 config maps, `vote` and `vote-dev`, which should look rather like this, but with placeholders filled in:
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: <name of cm>
+  namespace: vote
+data:
+  DB_URL: mongodb://<user>:<password>@tide.csh.rit.edu/vote?ssl=true
+  NODE_ENV: <env>
+  REACT_APP_BASE_API_URL: https://vote.csh.rit.edu
```

Comment:
For these values does the react app still just ignore this and pull from the .env for whatever reason? we probably need to fix that so that it actually will point to the dev service for dev

Reply:
Yeah I meant to ask you about that. Does dotenv get overridden by the actual environment? Looks like vote.cs.house is a wee bit broken, if I simulate loggint in as evals it's getting stuck. If flashes logged in and then goes back to redirecting.

Reply:
So the .env file was part of the boilerplate and I don't really know where it pulls it in off the top of my head but I could look into that at some point. When we tried to remove it completely the variables were "undefined", so I don't think it's getting the actual environment at all for some reason, but I'm not sure why

Reply:
Alright, I'm gonna switch to homework for the rest of today, but we'll need to fix that before we're ready for 1.0.0 I think. I've set the dev build to build off this branch, so we can do a bit more testing on OKD if we need.